app-requirements.txt

```diff
@@ -0,0 +1,78 @@
+appdirs==1.4.4 ; python_version >= "3.9" and python_version < "3.13"
```

Comment:
Just a note, the pip generator tool can't yet parse python_version syntax. But you'll know the target python version beforehand from the SDK.

Reply:
k thx, it's just there as this is the poetry export output and I kept it for the other files